src/components/Button/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import styles from "./styles/styles.module.scss";
+
+const Button = ({ text, path, type }) => {
+  let buttonStyle = styles.primary;
+
+  if (type === "success") {
+    buttonStyle = styles.success;
+  }
+
+  return (
+    <Link to={path}>
+      <button className={[buttonStyle, styles.button].join(" ")}>{text}</button>
+    </Link>
+  );
+};
+
+export default Button;
```

Comment:
[This comment](https://github.com/mim24sd/survey/pull/2/files#r1064529010) will change the above code a bit.

Reply:
yes